dataverse-webapp/src/main/webapp/resources/js/owl.carousel.js

```diff
@@ -1112,6 +1112,10 @@ if (typeof Object.create !== "function") {
             base.$elem.on("owl.jumpTo", function (event, item) {
                 base.jumpTo(item);
             });
+            // force reload to update the carousel item width
+            base.$elem.on("owl.reload", function (event, item) {
+                base.reload();
+            });
```

Comment:
Done.

Just as a side note, in the current version of the owlCarousel library the functionality I've added seems to be present out of the box (I haven't tried it in action, but it is listed in the docs). In case we ever upgrade, it's possible we might not need to modify the library at all.